### PR TITLE
Add task that tests the annotated jdk.

### PR DESCRIPTION
**Pull request by smillst**

### CODE REVIEW

File: checker/build.gradle [hunk 1]
```diff
@@ -382,6 +382,12 @@ task jtregJdk11Tests(dependsOn: ':downloadJtreg', group: 'Verification') {
 
     }
 }
+
+task testAnnotatedJdk {
+    description 'Runs some of the tests that use the annotated jdk.'
```
Comment (mernst) [inline checker/build.gradle:387]: Is this only some of the tests that use the annotated JDK?  Or is this some of the tests -- all those that use the annotated JDK?

Comment (mernst) [reply checker/build.gradle:387]: Nit: capitalize JDK.

File: checker/build.gradle [hunk 2]
```diff
@@ -382,6 +382,12 @@ task jtregJdk11Tests(dependsOn: ':downloadJtreg', group: 'Verification') {
 
     }
 }
+
+task testAnnotatedJdk {
+    description 'Runs tests that use the annotated JDK. (This does not run all the tests that use the annotated JDK.)'
```
Comment (mernst) [inline checker/build.gradle:387]: What other tests exist?  Why does this task only run some of them?

**Comment by smillst** [comment]
We decided to run allTests instead.

**Pull request closed without merging**